### PR TITLE
Combine Python version readme badges and remove code fence indentation

### PR DESCRIPTION
**Pull request by janosh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,21 +53,20 @@ To install Procrustes using the conda package management system, install
...
-    conda create -n myenv python=3.6
-    conda activate myenv
+# Create and activate myenv conda environment (optional, but recommended)
+conda create -n myenv python=3.6
+conda activate myenv
 
-    # Install the stable release.
-    conda install -c theochem qc-procrustes
+# Install the stable release.
+conda install -c theochem qc-procrustes
```
Comment (FanwangM) [inline README.md:61]: Thank you for the PR.

I don't think this part is necessary as the original makrdown documentation and GitHub instructions didn't impose such an indentation. https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

File: README.md [hunk 2]
```diff
@@ -59,21 +53,20 @@ To install Procrustes using the conda package management system, install
...
 
 To install Procrustes with pip, you may want to create a
 [virtual environment](https://docs.python.org/3/tutorial/venv.html), and then:
 
-
 ```bash
-    # Install the stable release.
-    pip install qc-procrustes
+# Install the stable release.
+pip install qc-procrustes
```
Comment (FanwangM) [inline README.md:69]: The same as above.

File: README.md [hunk 3]
```diff
@@ -44,12 +39,11 @@ Dependencies
...
-* NumPy >= 1.21.5: http://www.numpy.org/
-* SciPy >= 1.5.0: http://www.scipy.org/
-* PyTest >= 5.3.4: https://docs.pytest.org/
-* PyTest-Cov >= 2.8.0: https://pypi.org/project/pytest-cov/
-
+* Python >= 3.6: <http://www.python.org/>
+* NumPy >= 1.21.5: <http://www.numpy.org/>
+* SciPy >= 1.5.0: <http://www.scipy.org/>
+* PyTest >= 5.3.4: <https://docs.pytest.org/>
+* PyTest-Cov >= 2.8.0: <https://pypi.org/project/pytest-cov/>
```
Comment (FanwangM) [inline README.md:46]: GitHub takes care of rendering the urls automatically, https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls. Can you explain why you do this? Thank you.

**Review by FanwangM — CHANGES_REQUESTED** [review]
Thank you for the PR, which makes the package nicer. I have put some comments and we can merge this PR after addressing them.

File: README.md (hunk 1 above)
Comment (janosh) [reply README.md:61]: Do you have a reason for putting 4 spaces in front of a shell command? If not, unnecessary white space is best avoided.

File: README.md (hunk 3 above)
Comment (janosh) [reply README.md:46]: I didn't. Just the result of auto-formatting on save from [`markdownlint`](https://github.com/DavidAnson/markdownlint). The reason it does this (I assume) is that auto-converting URLs into HTML anchor tags is not part of the markdown spec. I.e. not all markdown parsers do this, GFM is just one variant that does.

Comment (FanwangM) [reply README.md:46]: This is a generalized solution. I am going to merge it soon. Thanks for the contribution! @janosh

**Review by FanwangM — APPROVED** [review]

**Pull request merged**